src/lib/ruby-generator/video.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Define Ruby code generator for Video Blocks
+ * @param {RubyGenerator} Generator The RubyGenerator
+ * @return {RubyGenerator} same as param.
+ */
+export default function (Generator) {
+    Generator.videoSensing_whenMotionGreaterThan = function (block) {
+        block.isStatement = true;
+        const rh = Generator.valueToCode(block, 'REFERENCE', Generator.ORDER_NONE) || 0;
+        return `${Generator.spriteName()}.when(:greater_than, self.video_motion, ${rh}) do\n`;
```

Comment:
内部で video_motion の値を参照するようにします。whenに渡す第1引数の名称を変更します。

```suggestion
        return `${Generator.spriteName()}.when(:video_motion_greater_than, ${rh}) do\n`;
```